packages/forms/src/Components/FileUpload.php

```diff
@@ -137,6 +146,11 @@ public function uploadProgressIndicatorPosition(string | Closure | null $positio
         return $this;
     }
 
+    public function getAppendFiles(): bool
```

Comment:
This should be `shouldAppendFiles()`, and `$this->appendFiles` should be `$this->shouldAppendFiles`